tests/x11/gnote/gnote_undo_redo.pm

```diff
@@ -28,6 +28,11 @@ sub undo_redo_once {
     assert_screen 'gnote-new-note-1';
 }
 
+sub jump_to_toolbar {
+    send_key "ctrl-tab";        #jump to toolbar
+    send_key "ctrl-tab" if is_sle('15+') && get_var("WAYLAND");    #jump to toolbar for SLED15
+}
```

Comment:
I suggest extract the common part to `lib/x11test.pm` that all gnote modules can use it.
We already have some gnote functions start from https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/x11test.pm#L793
And for this common part, please extract same statements as many as possible.

Reply:
Updated

Reply:
From your wayland verification run, I could not find the `WAYLAND` var in the test.
Could you update the verification runs?

Reply:
Thanks for pointing this out. I used  another way to go back to "all notes interface".
Verification run on SLED15 X11, Wayland and SLED12SP3 were also updated.